### PR TITLE
Handle no DNS at system start

### PR DESCRIPTION
**Pull request by nm-mrt**
If there is no DNS available on system start the capwap-wtp will stop while evaluating the configuration. I fixed that by checking if the address was yet translated on every address access. If not a DNS request is started before continuing.

### CODE REVIEW

**Comment by RoadRunnr** [comment]
@nm-mrt it is not necessary to open a new pull request when you update you change set. The pull request will automatically update if you push additional change to your branch or force push an updated version of your change.

File: src/common/capwap_network.h [hunk 1]
```diff
@@ -20,6 +21,12 @@ union sockaddr_capwap {
 	struct sockaddr_storage ss;
 };
 
+struct addr_capwap {
+	char fqdn[CAPWAP_MAX_FQDN_SIZE];
+	union sockaddr_capwap socket;
```
Comment (RoadRunnr) [inline src/common/capwap_network.h]: calling the sockaddr variable ```socket``` is a bit misleading, could we name it sockaddr instead?

File: src/common/capwap_network.h [hunk 2]
```diff
@@ -11,6 +11,7 @@
 /* */
 #define CAPWAP_MACADDRESS_EUI48_BUFFER		18
 #define CAPWAP_MACADDRESS_EUI64_BUFFER		24
+#define CAPWAP_MAX_FQDN_SIZE                128
```
Comment (RoadRunnr) [inline src/common/capwap_network.h]: max FQDN should be 256

Comment (thz) [inline src/common/capwap_network.h]: please change this to 256 according to RFC1035 (2.3.4.  Size limits)
(then we do not need any dyanmic allocation at all)

**Comment by nm-mrt** [comment]
Please check my changes - I don't have access to my build environment before start of next week.

**Comment by thz** [comment]
thank you! i merged it into master and also to v1.1.x branch

**Pull request merged**